tutoraspects/templates/openedx-assets/assets/datasets/monitoring.superset_action_log.yaml

```diff
@@ -0,0 +1,192 @@
+_file_name: monitoring.superset_action_log.yaml
+table_name: superset_action_log
+main_dttm_col: null
+description: null
+default_endpoint: null
+offset: 0
+cache_timeout: null
+schema: {{SUPERSET_DB_METADATA_NAME}}
+sql: "select\nl.dttm as action_date,\ncase \n  when lower(l.action)\
```

Comment:
note: [original query](https://github.com/hometogo/hometogo-data-code-snippets/blob/0db5a76aa9a814a93223e2932ed081a7750e80c9/superset/usage_dashboard/dashboard_export/datasets/SuperSet_Metadata/monitoring.superset_action_log.yaml#L8) used PostgreSQL's `date_trunc('day', l.dttm) as action_date` here.

MySQL's equivalent is `extract(day from l.dttm) as action_date`, but this caused errors in the charts, so I left the `l.dttm` datetime field as-is.